app/partials/buy-sell-select-partner.pug

```diff
@@ -36,11 +36,11 @@ div
               .item
                 input(type="email" ng-model="email" required)
             button.btn.button-primary.full-width.mt-20(translate="EXCHANGE.NOTIFY_ME_WHEN_AVAIL" type="submit")
-    .ml-30.width-40.hidden-xs(ng-if="!hasDismissedBuyIntro() && (country.Code === 'US' || $parent.country.Code === 'US')")
+    .ml-30.width-40.hidden-xs(ng-show="!blacklisted")
```

Comment:
Can we add a check to for if they canBuy and canSell with the partner that is selected?